pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -25,25 +25,43 @@ public PythonVersion(String version) {
         this.version = version;
     }
 
+    private String getVersionPart(int position) {
+        String[] parts = this.version.split("\\.");
```

Comment:
No need for `this.` here. There are no other arguments or variables named `version`.
These two lines can be reduced to `return version.split("\\.")[position];`
Could you move this private method down below the public API?